.gitignore

```diff
@@ -0,0 +1 @@
+login.py
```

Comment:
What is this file?

Reply:
config.py used to be named login.py. Changed.